### PR TITLE
chore: bump wrappers version to 0.2.0

### PR DESCRIPTION
**Pull request by burmecia**
## What kind of change does this PR introduce?

- upgrade wrappers from v0.1.19 to v0.2.0

## Additional context

This release will be announced in LWX.

## Action Items

- [ ] **New extension releases** were Checked for any breaking changes
- [ ] **Extensions compatibility** Checked
    * Proceed to [extensions compatibility testing](#extensions-compatibility-testing), mark as done after everything is completed
- [ ] **Backup and Restore** Checked
    * Proceed to [backup testing](#backup-testing) while extensions are enabled
        - After every restore, re-run the tests specified at point [3.1](#extensions-compatibility-testing)

### Extensions compatibility testing

1. Enable every extension
    1. Check Postgres’ log output for any error messages while doing so
        1. This might unearth incompatibilities due to unsupported internal functions, missing libraries, or missing permissions
2. Disable every extension
    1. Check Postgres’ log output for any cleanup-related error messages
3. Re-enable each extension
    1. Run basic tests against the features they offer, e.g.:
        1. `pg_net` - execute HTTP requests
        2. `pg_graphql` - execute queries and mutations
        3. …to be filled in

### Backup Testing

Follow the testing steps steps for all the following cases:

- Pause on new Postgres version, restore on new Postgres version
- Pause on older Postgres version, restore on new Postgres version
- Run a single-file backup backup, restore the backup

#### Testing steps

1. Generate dummy data 
    * the ‘Countries’ or ‘Slack clone’ SQL editor snippets are decent datasets to work with, albeit limited
2. Save a db stats snapshot file
    * Do this by running `supa db-stats gather -p <project_ref>`
3. Backup the database, through pausing the project, or otherwise
4. Restore the backup, through unpausing the project or cli
5. Check the data has been recovered successfully
    1. Visual checks/navigating through the tables works
    2. Run `supa db-stats verify` against the project and the previously saved file

### CODE REVIEW

File: common.vars.pkr.hcl [hunk 1]
```diff
@@ -1 +1 @@
-postgres-version = "15.1.0.137"
+postgres-version = "15.1.0.138-wrappers"
```
Comment (darora) [inline common.vars.pkr.hcl]: We're no longer merging suffixes to develop - you can build an image on this branch and test if needed

**Review by darora — COMMENTED** [review]
Can you confirm that you've tested pg_upgrades and pause/restores from the older versions?

File: common.vars.pkr.hcl (hunk 1 above)
Comment (burmecia) [reply common.vars.pkr.hcl]: sure

**Comment by burmecia** [comment]
> Can you confirm that you've tested pg_upgrades and pause/restores from the older versions?

Yes, I've tested pause/restore on staging and it works fine.

**Comment by darora** [comment]
@pcnc is testing pg_upgrades on staging possible with a custom AMI? Or does it need to be done with a local dev env?

**Review by darora — DISMISSED** [review]
@burmecia has confirmed off-channel that pg_upgrade is fine

**Comment by burmecia** [comment]
@darora I've tested pg_upgrade and it is working fine.

**Review by darora — APPROVED** [review]

**Pull request merged**